docs/ide/features.rst

```diff
@@ -0,0 +1,93 @@
+Language features
```

Comment:
I think this should be part of `index.rst`

Is not soo much content, so need for extra files.